components/MealDetailCircle.js

```diff
@@ -4,19 +4,27 @@
  *******************************************/
 
 import React from "react";
-import { StyleSheet, View, Text } from "react-native";
+import { StyleSheet, View, Text, ColorPropType } from "react-native";
```

Comment:
Where do you use this component? "ColorPropType"

Reply:
it happened automatically by mistake